tests/integration/template_complete_run.cfg

```diff
@@ -66,19 +66,13 @@ years = "1850:1854:2",
   mapping_file = ""
   vars = "RIVER_DISCHARGE_OVER_LAND_LIQ"
 
-[tc_analysis]
-active = True
-scratch = "#expand scratch#"
-walltime = "00:30:00"
-years = "1850:1854:2",
-
 [e3sm_diags]
 active = True
 grid = '180x360_aave'
 ref_final_yr = 2014
 ref_start_yr = 1985
 # TODO: this directory is missing OMI-MLS
-sets = "lat_lon","zonal_mean_xy","zonal_mean_2d","polar","cosp_histogram","meridional_mean_2d","enso_diags","qbo","diurnal_cycle","annual_cycle_zonal_mean","streamflow", "zonal_mean_2d_stratosphere", "tc_analysis",
+sets = "lat_lon","lat_lon_land"
```

Comment:
I think it might be more straightforward to start a new e3sm_diags task for "lat_on_land", for instance the test_data_path needs to be pointed to separate land files. And right now because there aren't many land obs in e3sm_diags. the lat_lon_land is used mostly for model vs model comparison.